jobs/async-upload/job/entrypoint.py

```diff
@@ -38,15 +38,12 @@ def main() -> None:
         perform_download(client, config)
 
         # Queue up model registration
-        registered_model = perform_model_registration(client, config)
-
-        # KServe Modelcars compatibility is handled within perform_upload()
-        # For OCI destinations, it automatically creates the required /models directory structure
-        # and prepares the model files in a KServe Modelcars-compatible format
+        set_artifact_pending(client, config)
```

Comment:
I think the pending state is set by the Dashboard per ADR instruction, but happy to keep this 'to make sure'